esmvaltool/utils/cmorize_obs.py

```diff
@@ -0,0 +1,273 @@
+"""
+Run the CMORization module as a utility executable.
+
+This utility allows the user to call and execute CMOR reformatting
+scripts (support for NCL and Python at the moment), that will use
+two I/O variables passed by this utility: an input directory as
+specified in config-user.yml by the RAWOBS key, and an output dir
+created in the form of output_dir/CMOR_DATE_TIME/TierTIER/DATASET.
+The user can specify a list of DATASETS that the CMOR reformatting
+can by run on by using -o (--obs-list-cmorize) command line argument.
+The CMOR reformatting scripts are to be found in:
+esmvaltool/cmor/cmorizers/obs
+
+    Usage
```

Comment:
Manually typing the Usage section is a bad idea, as it will get out of sync with the actual options available. If you just provide the `help='Some explanation'` argument to the `parser.add_argument` method, `argparse` will create this section for you when you run `mytool --help`.